core/components/atoms/_chip/__tests__/_chip.test.tsx

```diff
@@ -2,6 +2,8 @@ import * as React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import { testHelper, filterUndefined, valueHelper, testMessageHelper } from '@/utils/testHelper';
 import { GenericChip, GenericChipProps as Props } from '../../_chip';
```

Comment:
Please update imports to use root path instead of relative imports. 
`import { GenericChip, GenericChipProps as Props } from '../../_chip';` should be import { GenericChip, GenericChipProps as Props } from '@/components/atoms/_chip';